docs/python/static/doctools.js

```diff
@@ -0,0 +1,315 @@
+/*
+ * doctools.js
```

Comment:
It will be automatically generated whenever we do a sphinx-build. So I suppose if the `alabaster` theme is updated then this will different when it's generated. This is effectively the same as what would happen with `readthedocs` except it's just checked into source control.

Reply:
👍